[2025-08-22T13:47:29.368+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-22T13:47:29.412+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: orders_pipeline.create_external_table_if_not_exists manual__2025-08-22T13:46:52+00:00 [queued]>
[2025-08-22T13:47:29.425+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: orders_pipeline.create_external_table_if_not_exists manual__2025-08-22T13:46:52+00:00 [queued]>
[2025-08-22T13:47:29.426+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-08-22T13:47:29.450+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): create_external_table_if_not_exists> on 2025-08-22 13:46:52+00:00
[2025-08-22T13:47:29.458+0000] {standard_task_runner.py:64} INFO - Started process 63 to run task
[2025-08-22T13:47:29.464+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'orders_pipeline', 'create_external_table_if_not_exists', 'manual__2025-08-22T13:46:52+00:00', '--job-id', '154', '--raw', '--subdir', 'DAGS_FOLDER/orders_pipeline.py', '--cfg-path', '/tmp/tmpn_hwaeor']
[2025-08-22T13:47:29.467+0000] {standard_task_runner.py:91} INFO - Job 154: Subtask create_external_table_if_not_exists
[2025-08-22T13:47:29.583+0000] {task_command.py:426} INFO - Running <TaskInstance: orders_pipeline.create_external_table_if_not_exists manual__2025-08-22T13:46:52+00:00 [running]> on host b8d451ab62d9
[2025-08-22T13:47:29.798+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='orders_pipeline' AIRFLOW_CTX_TASK_ID='create_external_table_if_not_exists' AIRFLOW_CTX_EXECUTION_DATE='2025-08-22T13:46:52+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-22T13:46:52+00:00'
[2025-08-22T13:47:29.800+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-22T13:47:29.824+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-08-22T13:47:29.826+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'set -Eeuo pipefail -x; \nSQL="""\nCREATE EXTERNAL TABLE IF NOT EXISTS analytics.orders_raw (\n  order_id    int,\n  order_date  string,\n  customer    string,\n  amount      double\n)\nROW FORMAT SERDE \'org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe\'\nWITH SERDEPROPERTIES (\'serialization.format\' = \',\', \'field.delim\' = \',\')\nLOCATION \'s3://$S3_BUCKET/raw-data/orders/\'\nTBLPROPERTIES (\'skip.header.line.count\'=\'1\');\n"""\naws athena start-query-execution   --query-string "$SQL"   --query-execution-context "Database=$ATHENA_DB"   --work-group primary >/dev/null']
[2025-08-22T13:47:29.844+0000] {subprocess.py:86} INFO - Output:
[2025-08-22T13:47:29.846+0000] {subprocess.py:93} INFO - + SQL='
[2025-08-22T13:47:29.847+0000] {subprocess.py:93} INFO - CREATE EXTERNAL TABLE IF NOT EXISTS analytics.orders_raw (
[2025-08-22T13:47:29.847+0000] {subprocess.py:93} INFO -   order_id    int,
[2025-08-22T13:47:29.848+0000] {subprocess.py:93} INFO -   order_date  string,
[2025-08-22T13:47:29.848+0000] {subprocess.py:93} INFO -   customer    string,
[2025-08-22T13:47:29.848+0000] {subprocess.py:93} INFO -   amount      double
[2025-08-22T13:47:29.849+0000] {subprocess.py:93} INFO - )
[2025-08-22T13:47:29.849+0000] {subprocess.py:93} INFO - ROW FORMAT SERDE '\''org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe'\''
[2025-08-22T13:47:29.849+0000] {subprocess.py:93} INFO - WITH SERDEPROPERTIES ('\''serialization.format'\'' = '\'','\'', '\''field.delim'\'' = '\'','\'')
[2025-08-22T13:47:29.850+0000] {subprocess.py:93} INFO - LOCATION '\''s3://pt-data-lab/raw-data/orders/'\''
[2025-08-22T13:47:29.850+0000] {subprocess.py:93} INFO - TBLPROPERTIES ('\''skip.header.line.count'\''='\''1'\'');
[2025-08-22T13:47:29.851+0000] {subprocess.py:93} INFO - '
[2025-08-22T13:47:29.851+0000] {subprocess.py:93} INFO - + aws athena start-query-execution --query-string '
[2025-08-22T13:47:29.851+0000] {subprocess.py:93} INFO - CREATE EXTERNAL TABLE IF NOT EXISTS analytics.orders_raw (
[2025-08-22T13:47:29.851+0000] {subprocess.py:93} INFO -   order_id    int,
[2025-08-22T13:47:29.852+0000] {subprocess.py:93} INFO -   order_date  string,
[2025-08-22T13:47:29.852+0000] {subprocess.py:93} INFO -   customer    string,
[2025-08-22T13:47:29.853+0000] {subprocess.py:93} INFO -   amount      double
[2025-08-22T13:47:29.853+0000] {subprocess.py:93} INFO - )
[2025-08-22T13:47:29.853+0000] {subprocess.py:93} INFO - ROW FORMAT SERDE '\''org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe'\''
[2025-08-22T13:47:29.854+0000] {subprocess.py:93} INFO - WITH SERDEPROPERTIES ('\''serialization.format'\'' = '\'','\'', '\''field.delim'\'' = '\'','\'')
[2025-08-22T13:47:29.854+0000] {subprocess.py:93} INFO - LOCATION '\''s3://pt-data-lab/raw-data/orders/'\''
[2025-08-22T13:47:29.855+0000] {subprocess.py:93} INFO - TBLPROPERTIES ('\''skip.header.line.count'\''='\''1'\'');
[2025-08-22T13:47:29.855+0000] {subprocess.py:93} INFO - ' --query-execution-context Database=analytics --work-group primary
[2025-08-22T13:47:32.783+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-08-22T13:47:32.784+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-22T13:47:32.825+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=orders_pipeline, task_id=create_external_table_if_not_exists, run_id=manual__2025-08-22T13:46:52+00:00, execution_date=20250822T134652, start_date=20250822T134729, end_date=20250822T134732
[2025-08-22T13:47:32.878+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2025-08-22T13:47:32.921+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-08-22T13:47:32.924+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
